backend/spotihue/spotihue.py

```diff
@@ -471,15 +459,16 @@ def retrieve_available_lights_rgb(self) -> dict:
             dict: A dictionary of light names and their RGB values, or an
```

Comment:
Since we aren't maintaining both a `retrieve_available_lights()` and a `retrieve_available_lights_rgb()`, I think we could keep this named `retrieve_available_lights()` -  I don't think we need to rename the function based on what attributes of the lights' we choose to return (ex. retrieve_available_lights_name() or retrieve_available_lights_id()).